src/query/pipeline/sources/src/input_formats/impls/input_format_csv.rs

```diff
@@ -57,14 +59,46 @@ impl InputFormatCSV {
         column_index: usize,
         schema: &TableSchemaRef,
         default_values: &Option<Vec<Scalar>>,
+        empty_filed_as: &EmptyFieldAs,
     ) -> std::result::Result<(), FileParseError> {
         if col_data.is_empty() {
             match default_values {
                 None => {
+                    // query
                     builder.push_default();
                 }
                 Some(values) => {
-                    builder.push(values[column_index].as_ref());
+                    let field = &schema.fields()[column_index];
+                    // copy
+                    match empty_filed_as {
```

Comment:
Does `empty_filed_as` not work without `default_values`?

Reply:
thanks,   I will do it in next pr.

1. default_values = None only when querying stage (if I am not wrong), not copy.  (no dest table => no field_defaults),  empty_filed_as is ignored for now
2. when querying csv,  all field `$1 $2 ...` are nullable string.  and query not need to be to strict?  the consideration is a little different.  I